cfme/cloud/provider/__init__.py

```diff
@@ -73,10 +74,11 @@ def is_displayed(self):
     including_entities = View.include(VMEntities, use_parent=True)
 
 
+@attr.s(hash=False)
```

Comment:
Question: Why we need to use hash=False here?

Reply:
it is related to the way how attr based objects are compared. This override removes some comparison issues.